index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <link rel="stylesheet" href="style.css">
+  <title>Yuriy's Portfolio</title>
+</head>
+
+<body>
+  <div class="container">
+    <header>
+      <span class="logo">My Logo</span>
+      <nav>
+        <i class="fas fa-bars hidden"></i>
+        <ul class="menu">
+          <li class="menu-item">Portfolio</li>
+          <li class="menu-item">About</li>
+          <li class="menu-item">Contact</li>
+        </ul>
+      </nav>
+    </header>
+    <section class="headline">
+      <div class="main">
+        <h1 class="title">I'm Yuriy<br>Glad to see you!</h1>
+        <p class="main-text">I’m a software developer! I can help you build a product, feature or website. Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
```

Comment:
- [ ] As requested by the first reviewer Kindly adjust the background image of the mobile view to match the original design.
![image](https://user-images.githubusercontent.com/43698511/138263421-9bcdcc8a-2bb0-4fc4-ad31-c71ddfcf6438.png)
![image](https://user-images.githubusercontent.com/43698511/138263498-a4ae8b2d-53a8-4e92-a641-be746ed8d626.png)